src/runtime/RegExpObject.h

```diff
@@ -110,6 +110,11 @@ class RegExpObject : public Object {
         return m_source;
     }
 
+    String* RegExpOptionString()
```

Comment:
Could you change function name from RegExpOptionString to regexpOptionString or optionString?